src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -9,6 +9,13 @@ import { serialize, deserialize } from "@azure-tools/codegen";
 import { Session, startSession, Host, Channel } from '@azure-tools/autorest-extension-base';
 import { ToSnakeCase } from '../../utils/helper';
 
+function sortOperationByAzCommand(model: CodeModel) {
```

Comment:
just make it a private metod of CodeModelCliImpl

Reply:
changed now